vllm/utils.py

```diff
@@ -28,6 +28,13 @@
 T = TypeVar("T")
 logger = init_logger(__name__)
 
+try:
+    import intel_extension_for_pytorch as ipex  # noqa: F401
+    _import_ipex = True
+except ImportError as e:
+    logger.warning("Import Error for IPEX: %s", e.msg)
+    _import_ipex = False
```

Comment:
Why do we have to import `ipex` in this file? Can we do it only for the XPU backend? We'd like to avoid printing this warning for the NVIDIA GPU users.

Reply:
move to `is_xpu` method